app/Resources/views/markdown/pr_table_errors.md.twig

```diff
@@ -1,16 +1,21 @@
 <!-- PR_TABLE_DESCRIPTION_ERROR -->
 Hi, thanks for this contribution!
 
-Your pull request description seems to be incomplete or malformed:
+I found some issues with the Pull Request description:
```

Comment:
I prefer the notion of 'incomplete description' rather than 'issues', it sounds more positive.

Reply:
The problem that I see with the notion of "incomplete" is that it suggests that _the job is not complete_ and the that you _should have_, which is a way of saying "you didn't do it right". 

Using "issues" is, in my perspective, a way of pointing out a problem in a more detached, subjective approach. In addition, the use of "I" renforces the modal nature (aka subjective perception) of Prestonbot's discourse, as opposed to a global judgment about the contributor's work ("there's a problem" vs "you did it wrong"). Also, notice how it's no longer directing the judgment towards "their" PR and instead it speaks of "the" PR.

The main reason I rephrased this was due to issue linking; since it's actually optional, I thought it should be expressed as a suggestion instead of scolding the contributor for something they are free not to do.

If you're not comfortable with the word "issues", we can use "problems" as well.